### PR TITLE
Add core number to web stats

### PR DESCRIPTION
**Pull request by kevindweb**
Adds a core number label in the web stats view for each running network function.

## Summary:
Since getting access to the `nf_info->core`, we wanted to add this information to our web statistics. In the future the core mapping page will be updated, showing all NFs and manager on their specific cores. Building the web app has to delete and recreate many files, which is why there are so many "changes". There are only 2 line changes from `/onvm/onvm_mgr/onvm_stats.c`

**Usage:**
See the web stats [documentation](https://github.com/sdnfv/openNetVM/blob/develop/onvm_web/README.md) for information about running the React website. 

| This PR includes         |          |
| ------------------------ | -------- |
| Resolves issues          | 
| Breaking API changes     |
| Internal API changes     |
| Usability improvements   |  
| Bug fixes                |
| New functionality        |
| New NF/onvm_mgr args     | 
| Changes to starting NFs  |  
| Dependency updates       | 
| Web stats updates        | 👍

## Merging notes:
 - Dependencies: None  

**TODO before merging :**
 - [x] PR is ready for review

## Test Plan:
Run the web stats with `./go.sh 0,1,2 1 0xF8 -s web` and run speed tester nfs to see if the core number affects the website. 

## Review: 
- Sanity checks, assigned to @koolzz  @Arjun-Vijay
  - Run make in /onvm and /examples directories
  - Code style, assigned to @koolzz  @Arjun-Vijay

- Run linter
  - Check everything style related
  - Code design, assigned to @koolzz  @Arjun-Vijay

- Check for memory leaks
  - Check that code is reusable
  - Code is clean, functions are concise
  - Verify that edge cases properly handled

- Performance, assigned to @koolzz  @Arjun-Vijay
  - Run Speed Tester NF, report performance.

- Documentation, assigned to @koolzz  @Arjun-Vijay
  - Check if the new changes are well documented, in both code and READMEs

### CODE REVIEW

**Comment by onvm** [comment]
> In response to PR creation

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> In response to PR creation

## CI Message

Run successful see results:
[Results from nimbnode30]
Median TX pps for Speed Tester: 35193103

## Linter Failed

examples/aes_decrypt/aes.c:523:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1024:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1028:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1081:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1087:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 5
examples/aes_encrypt/aes.c:523:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1024:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1028:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1081:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1087:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 5
examples/arp_response/arp_response.c:144:  Missing space before ( in switch(  [whitespace/parens] [5]
Total errors found: 1
examples/bridge/bridge.c:135:  Missing space before ( in if(  [whitespace/parens] [5]
examples/bridge/bridge.c:138:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/bridge/bridge.c:156:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 3
examples/flow_table/flow_table.c:103:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:198:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:216:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:224:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:231:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/flow_table.c:238:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 6
examples/flow_table/msgbuf.c:71:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:73:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:83:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:86:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:89:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:106:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:108:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:116:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:119:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:125:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:138:  Extra space before last semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]
examples/flow_table/msgbuf.c:140:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:149:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:157:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:159:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:162:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:169:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 17
examples/flow_table/sdn.c:99:  Extra space before last semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]
examples/flow_table/sdn.c:125:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:127:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:147:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:147:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/sdn.c:153:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/sdn.c:160:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/sdn.c:235:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:239:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:279:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:418:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:423:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:493:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:501:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:538:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:542:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:545:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:547:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:555:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:569:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 20
examples/flow_table/setupconn.c:82:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:89:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:94:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:102:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:110:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/setupconn.c:112:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/setupconn.c:122:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:140:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:149:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:156:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 10
examples/flow_tracker/flow_tracker.c:108:  Missing space before ( in switch(  [whitespace/parens] [5]
examples/flow_tracker/flow_tracker.c:210:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 2
examples/scaling_example/scaling.c:312:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 1
onvm/onvm_mgr/main.c:98:  Mismatching spaces inside () in while  [whitespace/parens] [5]
Total errors found: 1
onvm/onvm_mgr/onvm_args.c:166:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:166:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:172:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:172:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:328:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:352:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:357:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 7
onvm/onvm_mgr/onvm_init.c:222:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
Total errors found: 1
onvm/onvm_mgr/onvm_nf.c:182:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_nf.c:255:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 2
onvm/onvm_mgr/onvm_stats.c:183:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_stats.c:183:  Missing space before {  [whitespace/braces] [5]
Total errors found: 2
onvm/onvm_nflib/onvm_config_common.c:52:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_config_common.c:112:  Missing space before ( in for(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_config_common.c:112:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_config_common.c:417:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
onvm/onvm_nflib/onvm_config_common.c:433:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
onvm/onvm_nflib/onvm_config_common.c:487:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
Total errors found: 6
onvm/onvm_nflib/onvm_flow_dir.c:64:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_flow_dir.c:95:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:103:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:111:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:128:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:143:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:151:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:159:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:176:  Missing space before {  [whitespace/braces] [5]
Total errors found: 9
onvm/onvm_nflib/onvm_nflib.c:341:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:344:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:347:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:351:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:354:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:357:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:584:  Missing space before ( in switch(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:743:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:754:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:821:  Missing space before {  [whitespace/braces] [5]
Total errors found: 10
onvm/onvm_nflib/onvm_pkt_helper.c:313:  Missing space before ( in switch(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_pkt_helper.c:376:  Missing space before {  [whitespace/braces] [5]
Total errors found: 2

**Comment by koolzz** [comment]
Pull the latest Dev branch commits, there shouldn't be any linter issues

**Comment by koolzz** [comment]
@aaroncoplan is this were we need to rebuild the webstuff

**Comment by koolzz** [comment]
I see this rebuild the web stuff so all good

**Comment by AaronCoplan** [comment]
Based on the fact this PR has updates to files within the `onvm_web/web-build/` path, it's an indicator that @kevindweb has already run the build.  But yes we will want to run it one more time before merging to ensure that the files are up to date.

**Comment by kevindweb** [comment]
@onvm let's try again

**Comment by onvm** [comment]
> @onvm let's try again

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> @onvm let's try again

## CI Message

Run successful see results:
[Results from nimbnode30]
Median TX pps for Speed Tester: 35161576

## Linter Passed

**Comment by koolzz** [comment]
I clicked on this first, brace yourselves for some CI emails

@onvm gooo

**Comment by koolzz** [comment]
@onvm hello

**Comment by koolzz** [comment]
@onvm bump

**Comment by koolzz** [comment]
@onvm bump

**Comment by koolzz** [comment]
@onvm boomp

**Comment by koolzz** [comment]
@onvm bip bop

**Comment by koolzz** [comment]
@onvm pls

**Comment by koolzz** [comment]
@onvm go

**Comment by koolzz** [comment]
@onvm gogogo

**Comment by koolzz** [comment]
@onvm тык

**Comment by onvm** [comment]
> @onvm тык

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> @onvm тык

## CI Message

Error: ERROR: Script failed on nimbnode30


## Linter Passed

**Comment by koolzz** [comment]
@onvm good good again

**Comment by onvm** [comment]
> @onvm good good again

## CI Message

Your results will arrive shortly

**Comment by koolzz** [comment]
@onvm тык

**Comment by onvm** [comment]
> @onvm тык

## CI Message

Another CI run in progress, please try again in 15 minutes

**Comment by onvm** [comment]
> @onvm good good again

## CI Message

Run successful see results:
[Results from nimbnode30]
Median TX pps for Speed Tester: 35158836

## Linter Passed

**Comment by koolzz** [comment]
@onvm pls work I wanna sleep

**Comment by onvm** [comment]
> @onvm pls work I wanna sleep

## CI Message

Your results will arrive shortly

**Comment by koolzz** [comment]
@onvm last time

**Comment by onvm** [comment]
> @onvm last time

## CI Message

Your results will arrive shortly

**Comment by koolzz** [comment]
@onvm well now last time for sure

**Comment by onvm** [comment]
> @onvm well now last time for sure

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> @onvm well now last time for sure

## CI Message

Error: ERROR: Script failed on nimbnode30


## Linter Passed

File: docs/Debug.md [hunk 1]
```diff
@@ -1,57 +1,59 @@
-Debugging OpenNetVM
-==
+# Debugging OpenNetVM
```
Comment (koolzz) [inline docs/Debug.md]: A few doc changes are fine to mix in with other PRs, but you have this + install doc which is about 400 line changes. Can you move it to another PR?

**Comment by koolzz** [comment]
@kevindweb Could you look into adding a minify step to the CSS and JS files to the build? This would fix our problem with the build being too inflated (this pr alone is 55k line changes right now)

File: docs/Debug.md (hunk 1 above)
Comment (kevindweb) [reply docs/Debug.md]: Sorry I didn't see this, I'll fix it

**Comment by kevindweb** [comment]
Finally got minified files to work! @onvm check it out one last time

**Comment by onvm** [comment]
> Finally got minified files to work! @onvm check it out one last time

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> Finally got minified files to work! @onvm check it out one last time

## CI Message

Run successful see results:
[Results from nimbnode30]
Median TX pps for Speed Tester: 35211377

## Linter Passed
